octobot_backtesting/collectors/exchanges/exchange_collector.py

```diff
@@ -72,6 +77,23 @@ async def initialize(self):
         self.config[commons_constants.CONFIG_CRYPTO_CURRENCIES] = {"Symbols": {
             commons_constants.CONFIG_CRYPTO_PAIRS: self.symbols}}
 
+    @staticmethod
+    async def historical_ohlcv_collector(exchange_manager, symbol, time_frame, start_time, end_time):
+        reached_max = False
+        while start_time < end_time and not reached_max:
+            candles = await exchange_manager.exchange.get_symbol_prices(symbol, time_frame, since=int(start_time))
```

Comment:
This class didn't call the exchange before. Either its design is obsolete or this method shouldn't be placed here.

Reply:
👍 migrated into `get_historical_ohlcv` in octobot_trading